### PR TITLE
Implement message handler for dialout support.

### PR DESCRIPTION
**Pull request by fancycode**
See https://github.com/nextcloud/spreed/issues/10410 for the Talk side.

### CODE REVIEW

File: docs/standalone-signaling-api-v1.md [hunk 1]
```diff
@@ -936,6 +1004,13 @@ Message format (Client -> Server):
     }
 
 
+Phone sssions will have `type` set to `phone` in the additional user data
```
[the remark this answers is not in the document]
Comment (fancycode) [reply docs/standalone-signaling-api-v1.md]: Thanks for spotting this!

File: docs/standalone-signaling-api-v1.md [hunk 2]
```diff
@@ -793,6 +793,74 @@ Message format (Server -> Client, receive message)
...
+is available.
+
+These messages can be used to perform actions on clients that should only be
+possible by some users (e.g. moderators).
+
+Message format (Client -> Server, mute phone):
+
+    {
+      "id": "unique-request-id",
+      "type": "message",
```
Comment (nickvergessen) [inline docs/standalone-signaling-api-v1.md:809]: should be:
```suggestion
      "type": "control",
```
?

File: docs/standalone-signaling-api-v1.md [hunk 3]
```diff
@@ -793,6 +793,74 @@ Message format (Server -> Client, receive message)
...
+- `1`: mute speaking (i.e. phone can no longer talk)
+- `2`: mute listening (i.e. phone is on hold and can no longer hear)
+
+To unmute, a value of `0` must be sent.
+
+Message format (Client -> Server, hangup phone):
+
+    {
+      "id": "unique-request-id",
+      "type": "message",
```
Comment (nickvergessen) [inline docs/standalone-signaling-api-v1.md:832]:
```suggestion
      "type": "control",
```

File: docs/standalone-signaling-api-v1.md [hunk 4]
```diff
@@ -793,6 +793,74 @@ Message format (Server -> Client, receive message)
...
+          "type": "hangup"
+        }
+      }
+    }
+
+Message format (Client -> Server, send DTMF):
+
+    {
+      "id": "unique-request-id",
+      "type": "message",
```
Comment (nickvergessen) [inline docs/standalone-signaling-api-v1.md:848]: Also control? Wasn't able to test this

File: docs/standalone-signaling-api-v1.md (hunk 2 above)
Comment (fancycode) [reply docs/standalone-signaling-api-v1.md]: Copy-paste :see_no_evil:

File: docs/standalone-signaling-api-v1.md (hunk 4 above)
Comment (fancycode) [reply docs/standalone-signaling-api-v1.md]: Yeah, thanks for spotting this.

**Pull request merged**